rpm/gmp-droid.spec

```diff
@@ -24,9 +26,15 @@ Gecko Media Plugin for droidmedia codec support in Gecko based browsers
 %install
 %meson_install
 
+# create config oneshot
+mkdir -p $RPM_BUILD_ROOT%{_oneshotdir}
+cp %{SOURCE1} $RPM_BUILD_ROOT%{_oneshotdir}/
```

Comment:
You can use `install  -D -m 0755 %{SOURCE1} %{buildroot}/%{_oneshotdir}` instead.